### ISSUE TITLE
OL 5.0.2 & Angular 6 ERROR TypeError: "invalid assignment to const `n'"

### ISSUE THREAD

**Issue by mrfx** [openlayers/openlayers#8369]
My project works just fine at development node server, and it compiles without errors, but at production, compiled with `ng build --prod` after dragging a map I've got at js console:

ERROR TypeError: "invalid assignment to const `n'"

![obraz](https://user-images.githubusercontent.com/6052773/42509478-4d882e1e-844c-11e8-9b25-b178f3031242.png)

**Comment by mrfx** [comment]
I think it is some incompability with UglifyJS, because when I compile with `ng build --build-optimizer --aot` then everything works good without errors.

**Comment by hei-pa** [comment]
I have the same error without a `\n` issue. It seems to be a true assignment to a const value which does not blow up in non production builds.

```
ERROR TypeError: Assignment to constant variable.
    at x.Lfyu.x.animate (main.1367c427a61f29c9fdc4.bundle.js:1)
    at x.Lfyu.x.fit (main.1367c427a61f29c9fdc4.bundle.js:1)
    at e._next (7.a455f51cf1275afc4caa.chunk.js:1)
```

**Comment by cvenzin** [comment]
observed the same issue

**Comment by karasavm** [comment]
the same here. 
Ionic 4
Angular 6
ol 5.0.0

**Comment by Rolandocepeda** [comment]
Running into the same problem, production build compiles and map loads but breaks when dragging or zooming in. Don't see the problem when not using `--prod` flag.  Does anyone know a fix/workaround for this issue? 
Using angular 6
ol 5.0.0

![errorol](https://user-images.githubusercontent.com/18712573/44743785-0c852180-aac9-11e8-95e5-7fbc59bda8f3.png)

**Comment by ahocevar** [comment]
@Rolandocepeda Thanks for your report, I now see what this issue was. And is has already been fixed with #8503.
